sweepbatcher/sweep_batch.go

```diff
@@ -624,9 +645,18 @@ func (b *batch) publish(ctx context.Context) error {
 		return err
 	}
 
-	fee, err, coopSuccess = b.publishBatchCoop(ctx)
-	if err != nil {
-		b.log.Warnf("co-op publish error: %v", err)
+	if b.cfg.mixedBatch {
+		fee, err = b.publishMixedBatch(ctx)
+		if err != nil {
+			b.log.Warnf("Mixed batch publish error: %v", err)
+		} else {
+			coopSuccess = true
```

Comment:
I changed `publishMixedBatch` interface: now it returns `signSuccess bool` result, as well as `publishBatchCoop` function. Also I renamed `coopSuccess` local variable here to `signSuccess`, since it indicates success of signing (and all preceding actions), which may be fully non-cooperative in case of `publishMixedBatch`.